src/site/_data/i18n/authors.yml

```diff
@@ -1102,3 +1102,9 @@ dandclark:
     en: Dan Clark
   description:
     en: Developer on the Microsoft Edge Web Platform team
+
+lozandier:
+  title:
+    en: Kevin Lozandier
+description:
+  en: User Experience Engineer on the Google Responsible
```

Comment:
I think this is wrong, because `description` is at the top-level.

Also, is this text cut off?